ChangeLog.d/getentropy.txt

```diff
@@ -0,0 +1,2 @@
+Changes
+    * On recent enough versions of FreeBSD and DragonFlyBSD, the entropy module now uses the getrandom syscall instead of reading from /dev/urandom.
```

Comment:
From `ChangeLog.d/00README.md`: "Each entry starts with **three** spaces, an asterisk and a space. Continuation lines start with 5 spaces. **Lines wrap at 79 characters**."

```suggestion
   * On recent enough versions of FreeBSD and DragonFlyBSD, the entropy module
     now uses the getrandom syscall instead of reading from /dev/urandom.
```